mlrun/datastore/google_cloud_storage.py

```diff
@@ -28,28 +32,62 @@ class GoogleCloudStorageStore(DataStore):
     using_bucket = True
 
     def __init__(self, parent, schema, name, endpoint="", secrets: dict = None):
+        self._storage_client = None
+        self._storage_options = None
+        self.workers = 8
+        self.chunk_size = 32 * 1024 * 1024
```

Comment:
If you're not allowing for overriding it with a parameter, please move to class level.